src/app/app.js

```diff
@@ -1,32 +1,58 @@
 import express from 'express';
+import cookieSession from 'cookie-session';
 import helmet from 'helmet';
 import pinoMiddleware from 'express-pino-logger';
 import compression from 'compression';
 import staticGzip from 'express-static-gzip';
+import fetch from 'node-fetch';
+
+import auth from '../auth';
 import home from '../home';
 import orgs from '../orgs';
 import {pageNotFoundMiddleware, internalServerErrorMiddleware} from '../errors';
 import csp from './app.csp';
 
-export default function ({logger} = {}) {
+export default function (config) {
   const app = express();
 
-  if (logger) {
-    app.use(pinoMiddleware({logger}));
-  }
+  app.use(pinoMiddleware(config.logger));
+
+  app.use(cookieSession({
+    name: 'pazmin-session',
+    keys: [config.sessionSecret],
+    secure: config.isProduction,
```

Comment:
It's useful to be able to run the server locally with NODE_ENV=production to test minor build differences (like with minification/performance enhancements/whatever)... ie `npm run start:production`

by basing the `secure` on the NODE_ENV value I think this makes it currently impossible to run the "production" build locally.

Perhaps an `INSECURE=true` option to disable https-only cookies (and maybe other things like it) could work instead, so we could modify the `npm start` helper to set the `INSECURE=true` flag and maintain the ability to run the production build.

Reply:
Personally, I dislike the fact we need to pass bunch of env variables on start xD

```sh
OAUTH_AUTHORIZATION_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/authorize" OAUTH_TOKEN_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/token" OAUTH_CLIENT_ID="rafal-login" OAUTH_CLIENT_SECRET="secret" SERVER_ROOT_URL="http://localhost:3000" npm start
```

I'm use to `.env` files, but I don't want to introduce that here as I know people would disagree with me.

I agree with your point though, will do.

Reply:
Given the nature of this app we can't really avoid it ... however technically we can get the AUTH_ENDPOINT and TOKEN_ENDPOINT from the CF_API_ENDPOINT via /v2/info ... so that might make things a bit tidyier